app/views/spam2/_spam.html.erb

```diff
@@ -58,9 +58,9 @@
   <a class="btn btn-light border border-curve text-secondary mr-auto font-weight-bold" id="info-btn" data-toggle="collapse" href="#info-div" role="button" aria-expanded="false" aria-controls="info-div">
     Info <i class="fa fa-caret-down text-secondary"></i>
   </a>
-  <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_comments" && params[:action] != "_spam_flags" && params[:action] != "_spam_users"%>
-    <div class=" d-inline-flex"><a href="/spam2/filter/unmoderated/30"  class="badge badge-pill badge-light text-secondary mx-2 my-3"><%= Node.where(status: 4).length %> Unmoderated</a></div>
-    <div class=" d-inline-flex"><a href="/spam2/flags/filter/all/30" class="badge badge-pill badge-light text-secondary mx-2 my-3" ><%= Node.where('flag > ?', 0).length %> Flagged</a></div>
+  <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_comments" && params[:action] != "_spam_users"%>
+    <div class=" d-inline-flex"><i class="fa fa-sticky-note text-info mt-3 ml-5" data-toggle="tooltip" data-placement="left" title="Unmoderated Nodes"></i><a href="/spam2/filter/unmoderated/30"  class="badge badge-pill badge-light text-info mx-2 my-3"><%= @node_unmoderated_count %></a></div>
```

Comment:
@juliawithers  I have removed @node_unmoderated_count  in the controller. _Please keep these two lines_.

```
 <div class=" d-inline-flex"><a href="/spam2/filter/unmoderated/30"  class="badge badge-pill badge-light text-secondary mx-2 my-3"><%= Node.where(status: 4).length %> Unmoderated</a></div>
    <div class=" d-inline-flex"><a href="/spam2/flags/filter/all/30" class="badge badge-pill badge-light text-secondary mx-2 my-3" ><%= Node.where('flag > ?', 0).length %> Flagged</a></div>
```

Reply:
@keshavsethi I hope I understood correctly - I have replaced the lines. And I'm not entirely sure I did this the right way either... I have made the changes on my branch, but I'm unsure how to show the changes here in this pull request.